docs/book/src/Good Practices

```diff
@@ -0,0 +1,24 @@
+# Good Practices
```

Comment:
Also, can you add the section Good Practices under the reference in the left menu and link this one?